pkg/config/config.go

```diff
@@ -34,25 +32,15 @@ func ReadConfigFile(cfgPath string) (dig.Mapping, error) {
 }
 
 // RenderK0sConfig renders a k0s cluster configuration.
-func RenderK0sConfig(ctx context.Context) (*k0sconfig.ClusterConfig, error) {
-	bin := defaults.PathToEmbeddedClusterBinary("k0s")
-	cmd := exec.Command(bin, "config", "create")
-	output, err := cmd.Output()
-	if err != nil {
-		return nil, fmt.Errorf("unable to generate default config: %w", err)
-	}
-	var cfg k0sconfig.ClusterConfig
-	if err := k8syaml.Unmarshal(output, &cfg); err != nil {
-		return nil, fmt.Errorf("unable to unmarshal default config: %w", err)
-	}
+func RenderK0sConfig() *k0sconfig.ClusterConfig {
+	cfg := k0sconfig.DefaultClusterConfig()
```

Comment:
I'm a littttle skeptical of actually doing this
I think it'll be fine, but it couples the k0s images to the go.mod instead of the embedded binary

Reply:
I've changed our `make` command to pull that version of k0s so that we always have a match